tx.go

```diff
@@ -176,6 +178,21 @@ func TraceGeneric(ctx context.Context, name string) *Trace {
 	})
 }
 
+// TraceDatastore adds a datastore segment to the newrelic transaction, if one exists in the context.
+func TraceDatastore(ctx context.Context, table, operation, sql, rollupName string) *Trace {
```

Comment:
Yep, sql is the raw query string. The agent obfuscates the sql. I'm not sure what the rollupName is, but I assume is allows you to group similar queries together or something.